testkit/src/mill/testkit/BspServerUtil.scala

```diff
@@ -0,0 +1,184 @@
+package mill.testkit
+
+import ch.epfl.scala.{bsp4j => b}
+import com.google.gson.{Gson, GsonBuilder}
+import coursier.cache.CacheDefaults
+import mill.api.BuildInfo
+import mill.bsp.Constants
+import org.eclipse.{lsp4j => l}
+
+import java.util.concurrent.atomic.AtomicInteger
+import java.util.concurrent.{ExecutorService, Executors, ThreadFactory}
+
+import scala.jdk.CollectionConverters._
+import scala.reflect.ClassTag
+
+object BspServerUtil {
+
+  val updateFixtures = false
+
+  lazy val bsp4jVersion: String = sys.props.getOrElse("BSP4J_VERSION", ???)
```

Comment:
We should just use a `val` or `def` here unless there is some explicit reason we want it to be lazy (e.g. due to side effecting initialization and/or performance overhead)

Reply:
Sure